### DISCUSSION TITLE
Bandwidth hog

### DISCUSSION THREAD

**Discussion by ScratchMode** [jef/streetmerchant#438] [Ideas and feature requests]
I've run a network usage meter alongside this tool and measured about 70Mb a minute (98Gb a day) which would blow past a data cap from my ISP in 10 days. 

I already have Low-Bandwidth set to true.
Are there any other optimizations that can be made to reduce this overhead?

**Comment by jef** [comment]
This is because Chromium. We could probably get rid of images in the sites, but not sure how that would effect overall consistency of nvidia-snatcher.

https://github.com/jef/nvidia-snatcher/issues/164#issuecomment-696269885

**Comment by jxjftw** [comment]
You should probably just join a discord that is notifying already if you have a bandwidth cap. That or run NV snatcher on a cloud VM but that will get pricey.

**Comment by ThisIsPita** [comment]
Would it affect the ability to correctly detect stock status if we used a persistent path for the cache for every chrome process?

Think that would also help with [#378 ](https://github.com/jef/nvidia-snatcher/issues/378#issue-711128014) since there would only be one shared profile between all the processes.

**Comment by ScratchMode** [comment]
> You should probably just join a discord that is notifying already if you have a bandwidth cap. That or run NV snatcher on a cloud VM but that will get pricey.

Do you know a public discord running this particular stock checker? My interest is to cover as many bases as possible so I am already subscribe to @rtxbot, Nowinstock, stockinformer, Nvidia-Clerk and Gearinstock.

**Comment by jxjftw** [comment]
> > You should probably just join a discord that is notifying already if you have a bandwidth cap. That or run NV snatcher on a cloud VM but that will get pricey.
> 
> Do you know a public discord running this particular stock checker? My interest is to cover as many bases as possible so I am already subscribe to @rtxbot, Nowinstock, stockinformer, Nvidia-Clerk and Gearinstock.

https://discord.gg/atr is running a modified version of this, as well as some other stuff.

**Comment by Tiffys** [comment]
Mess with chromium's caching perhaps? Rather than forcefully loading all of every website's contents every visit most data could be cached with very small kb html data needing to be loaded out of the cache each time. So rather than 10.1MB loading Best Buy for instance maybe only 548kb.
